config/settings.yaml.test

```diff
@@ -3,3 +3,5 @@
 :require_ssl: false
 :fqdn: foreman.example.com
 :domain: example.com
+
+# Do not change this file, create config/settings.yaml.test.local instead.
```

Comment:
Could we put content of this file to a hash in `config/settings.rb` and remove this file from git?
This line would be helpful (for people to know what file to create for testing), but we could probably put it in the `settings.yml.example`?

Reply:
What exactly moving the contents to `config/settings.rb` brings us?

Reply:
The idea behind using a fixed `settings.yaml.test` is that you get a very predictable environment where we know exactly what's going to happen. Having a local test file can break that assumption.